server/src/services/pdfReports/templates/PdfReportTemplate.tsx

```
@@ -0,0 +1,223 @@
+      previous: previousOverview?.bounce_rate,
+      format: (v: number | null) => `${safeToFixed(v, 1)}%`,
+      invertGrowth: true,
+    },
+  ];
+
+  return (
+    <html>
+      <head>
+        <meta charSet="UTF-8" />
+        <style
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)